presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -551,7 +552,12 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         NEW_PARTITION_USER_SUPPLIED_PARAMETER,
                         "\"user_supplied\" parameter added to all newly created partitions",
                         null,
-                        true));
+                        true),
+                integerProperty(
```

Comment:
Do we want to expose this as a session option? Exposing it can make the users to override it, beats the purpose of limits.

Reply:
exactly. I think a configuration is good. session property will be override by users

Reply:
For record, had an offline discussion with Venki, it is okay for users to override given our target use cases. Elaborated the use cases in the updated description section